Team417_2020/src/main/java/org/firstinspires/ftc/team417_2020/AutoCompetition.java

```diff
@@ -0,0 +1,131 @@
+package org.firstinspires.ftc.team417_2020;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.robotcore.external.hardware.camera.WebcamName;
+import org.firstinspires.ftc.team417_2020.Resources.Toggler;
+import org.openftc.easyopencv.OpenCvCamera;
+import org.openftc.easyopencv.OpenCvCameraFactory;
+import org.openftc.easyopencv.OpenCvCameraRotation;
+
+@Autonomous(name="Competition Auto")
+public class AutoCompetition extends MasterAutonomous{
+
+    int allianceSide = 1;
+    Toggler rightBumper = new Toggler();
+
+    RingDetectionOpenCV ringDetector = new RingDetectionOpenCV();
+    OpenCvCamera webcam;
+    int numRings;
+
+
```

Comment:
This is a smaller formatting thing, but typically we don't want to have multiple blank lines. One should be enough.